gp-templates/translations.php

```diff
@@ -165,9 +174,20 @@
 		 *
 		 * This action is inside a DL element.
 		 *
+		 * @deprecated 2.1.0 Call gp_translation_set_sort_form instead
```

Comment:
Instead of having both filters in the template, what do you think about calling it via `gp_translation_set_sort_form`?
```
add_action( 'gp_translation_set_sort_form', function() {
    do_action( 'gp_translation_set_filters' );
}, 5 );
```